docs/source/markdown/podman-network-create.1.md

```diff
@@ -54,9 +55,9 @@ must be used with a *subnet* option.
 
 Set metadata for a network (e.g., --label mykey=value).
 
-#### **--macvlan**
+#### **--macvlan**=*device*
```

Comment:
I think we should just remove the documentation rather then telling users about it?

Reply:
The option can still be there, just no reason to document it.  That way existing users will continue to use it, but no new users will accidentally start using it.

Reply:
Or perhaps we can add a "deprecated" note to this and then say as of 4.0 it's being removed.

Reply:
I think deprecated options should still be documented. This change clearly states this and what you should use instead. I could add that it is being removed with podman 4.0 but I am not sure if there is any reason to remove it.

Reply:
I disagree, if we don't want people to use or rely on an option, we should make the option disappear.  4.0 would allow us to break their scripts(Although I would recommend against that).

We can document in the release notes that it is depracated, and print a warning when users use it.

Reply:
I removed the flag from the man page and emit a warning when --macvlan is used.